recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
+    - icu
     - jom  # [win]
     - jpeg 8*  # [not osx]
-    - libpng >=1.6.21,<1.7  # [not osx]
+    - libpng  # [not osx]
```

Comment:
You're probably getting fontconfig from defaults, not conda-forge.  Add the conda-forge channel and try again.